packages/jest-cli/src/TestRunner.js

```diff
@@ -67,7 +69,7 @@ class TestRunner {
     this._setupReporters();
   }
 
-  addReporter(reporter: BaseReporter) {
+  addReporter(reporter: Object) {
```

Comment:
Can this be a type for Reporter instances in `types/TestRunner.js` or something?